Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.yml

```diff
@@ -39,7 +39,7 @@ script:
       required: false
       secret: false
     - default: false
-      description: 'Limit the results to those that have only changed since this time. Format: YYYY-MM-DD'
+      description: 'Limit the results to those that have only changed since this time. Date format will be YYYY-MM-DD format or relational expressions like “7 days ago”.'
```

Comment:
```suggestion
      description: 'Limit the results to those that have only changed since this time. Date format will be YYYY-MM-DD or relational expressions like “7 days ago”.'
```